test/extensions/filters/http/kill_request/kill_request_filter_test.cc

```diff
@@ -102,6 +102,20 @@ TEST_F(KillRequestFilterTest,
   EXPECT_DEATH(filter_->decodeHeaders(request_headers_, false), "");
 }
 
+TEST_F(KillRequestFilterTest,
+       KillRequestDisabledWhenIsKillRequestEnabledReturnsFalseFromRouteLevelConfiguration) {
```

Comment:
These names for tests are kind of unreadably long. Can you replace them with a shorter name and then add a single line `//` comment above providing the intuition that is currently captured in the name? Here and elsewhere in these tests. Thanks.

Reply:
Done